### PR TITLE
Update compressed nfts

### PR DESCRIPTION
**Pull request by adpthegreat**
- Followed guidelines in contributing.md
- Refactored it to use newest dependencies 
- Replaced @metaplex- foundation/js with @metaplex-foundation-umi
- Replaced old methods with their corresponding umi methods 
- Replaced all old code snippets 
- Gave clearer variable names
- Removed redundant code 
- Ran prettier 

updated lesson code and created Unboxed PRs

[solution code](https://github.com/Unboxed-Software/solana-cnft-demo/pull/2)
[starter code](https://github.com/Unboxed-Software/solana-cnft-demo/pull/3)

### CODE REVIEW

**Comment by adpthegreat** [comment]
Ran prettier locally multiple times but it still fails in CI and I don't know the reason

File: content/courses/state-compression/compressed-nfts.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Compressed NFTs
 objectives:
-  - Create a compressed NFT collection using Metaplex's Bubblegum program
-  - Mint compressed NFTs using the Bubblegum TS SDK
-  - Transfer compressed NFTs using the Bubblegum TS SDK
+  - Create a compressed NFT collection using Metaplex’s Bubblegum program
```
Comment (mikemaccana) [inline content/courses/state-compression/compressed-nfts.md:4]: No need for smart quotes. Just use `'`

File: content/courses/state-compression/compressed-nfts.md [hunk 2]
```diff
@@ -13,121 +13,122 @@ description:
...
-anyone using NFTs at scale could cut costs by more than 1000x by using cNFTs
-over traditional NFTs.
-
-However, cNFTs can be tricky to work with. Eventually, the tooling required to
-work with them will be sufficiently abstracted from the underlying technology
-that the developer experience between traditional NFTs and cNFTs will be
-negligible. But for now, you'll still need to understand the low level puzzle
-pieces, so let's dig in!
+use a technique called
+[**State Compression**](https://solana.com/docs/advanced/state-compression) to
```
Comment (mikemaccana) [inline content/courses/state-compression/compressed-nfts.md:31]: Nice word adding the link for the term.

File: content/courses/state-compression/compressed-nfts.md [hunk 3]
```diff
@@ -13,121 +13,122 @@ description:
...
+to this process are:
 
 1. Take any piece of data
-2. Create a hash of this data
-3. Store this hash as a “leaf” at the bottom of the tree
-4. Each leaf pair is then hashed together, creating a “branch”
-5. Each branch is then hashed together
+2. Create a hash of the data
+3. Store the hash as a “leaf” at the bottom of the tree
+4. Hash pairs of leaves together, to create “branches”
```
Comment (mikemaccana) [inline content/courses/state-compression/compressed-nfts.md:64]: Good work, this is much clearer than the older content.

File: content/courses/state-compression/compressed-nfts.md [hunk 4]
```diff
@@ -13,121 +13,122 @@ description:
...
+5. Hash branches together
 6. Continually climb the tree and hash adjacent branches together
 7. Once at the top of the tree, a final ”root hash” is produced
-8. Store the root hash onchain as a verifiable proof of the data within each
-   leaf
+8. Store the root hash onchain as a verifiable proof that the data within each
+   leaf has not changed
```
Comment (mikemaccana) [inline content/courses/state-compression/compressed-nfts.md:69]: Also much clearer.

**Review by mikemaccana — CHANGES_REQUESTED** [review]
I haven't reviewed this fully, but it looks good so far - to speed up the review, can you please remove the smart quotes (just use `'`) and remove the other files from this PR? This will reduce the 'noise' and I'll then check out the remaining changes.

**Comment by adpthegreat** [comment]
Alright I'll do that , the other files from the PR included was because I was trying to fix the prettier CI issues, I'll probably just have to remove the last three or four commits

**Pull request closed without merging**